src/components/Accordion/index.tsx

```diff
@@ -0,0 +1,44 @@
+import Wrapper from '@components/Wrapper';
+import React from 'react';
+import picture from '@assets/Sign Out.svg';
+import { AccordionBody, AccordionTitle, Arrow } from './styles';
+
+const Accordion = () => {
+  const [toggle, setToggle] = React.useState(false);
+  const [heightEl, setHeightEl] = React.useState<string>();
+
+  const refHeight = React.useRef<HTMLDivElement>(null);
+
+  React.useEffect(() => {
+    if (refHeight.current)
+      setHeightEl(`${refHeight.current.scrollHeight + 20 + 16}px`);
```

Comment:
are you sure you need it? with current list of ready libraries when I see such code usually mean that something custom and bad happend

Reply:
I watched 10 min youtube video how to make accordeon :) I dont think that something bad will happened. It looks simple.

Reply:
why 20 + 16? why not 36?

Reply:
20 is top margin in accordeonBody, 16 is padding. It doesnt counted with scrollHeight property, so i need to apply it manually. I moved them to variables for better experience.